src/GovUk.Education.ExploreEducationStatistics.Admin/Migrations/ContentMigrations/20230706081603_MigrateMethodologyInternalReleaseNotes.sql

```diff
@@ -0,0 +1,11 @@
+﻿INSERT INTO [dbo].[MethodologyStatus] (Id, MethodologyVersionId, InternalReleaseNote,
+                                       ApprovalStatus, Created, CreatedById)
+SELECT NEWID() AS Id,
+       MV.Id AS MethodologyVersionId,
+       MV.InternalReleaseNote AS InternalReleaseNote,
```

Comment:
Will there be a follow-up piece of work to remove MethodologyVersion.InternalReleaseNote?

Reply:
I had considered this and initially thought we should leave it - but having given it a closer look, I think we should. I think it'll just involve a small change to `BuildMethodologyVersionViewModel` and a little unit test juggling.

I'll create a subtask.

Reply:
There is some question of how this should be implemented though. Right now, `InternalReleaseNote` only gets set when a release is approved. But a `MethodologyStatus` may contains a note when the methodology is switched to draft or higher review. So if we were to grab the latest `MethodologyStatus`, we will be changing the behaviour, I believe.